### ISSUE TITLE
Current solution breaks the original SuperCollider.app

### ISSUE THREAD

**Issue by rukano** [dathinaios/spacemacs-supercollider#2]
The current solution is to link the `sc` folder directly to the app support directoy, but this is a little bit destructive, since the normal SuperCollider.app won't compile (Duplicated Document.sc) class. I know there is this `'(sclang-library-configuration-file "path_to/emacs_cfg.yaml")` which I set in the custon-set-variables (in my old emacs system, not spacemacs)... But this doesn't seem to be working. In the config file is something like:

```yaml
includePaths:
    - /Applications/SuperCollider/SuperCollider.app/Contents/Resources/SCClassLibrary
    - /Library/Application Support/SuperCollider/Extensions
    - /Users/username/.emacs.d/private/supercollider/sc
excludePaths:
    []
postInlineWarnings: false
```

So the sclang started from emacs will compile these paths (including the emacs classes), and the normal SuperCollider.app won't notice anything... Do you have any idea why setting the path to library configuration file is not working? I've tried doing it in several places... in `.spacemacs`, in `packages.el`... no success... Only succeeded with your version, linking the `sc` folder directly to the extensions...

**Comment by dathinaios** [comment]
Yes, I just noticed this too...

The issue with the config might be a `spacemacs` specific problem. I've been having troubles with mapping keys and it seems that there are some things that need to be defined in the `local/sclang` files so that it works properly with `spacemacs`.

I will start by looking at the `sc` and `elisp` files to see if I can figure it out but my `elisp` knowledge is kind of.. zero at the moment.

**Comment by rukano** [comment]
ah! but that is at least something! i'm not an elisp pro either... but want to get this working, the rest of my spacemacs works like a charm, and just want to move also my favourite language and stop using the scide for my stuff too... ^_^ I'll try other solutions this weekend, and read some documentation on the spacemacs layers... aybe there's a hook/function where we can put these definitions...

**Comment by dathinaios** [comment]
Haha, I am on the same boat. I have moved to `spacemacs` for everything but still have to use `vim` for sc...

**Comment by dathinaios** [comment]
This should be now solved  (c6f24a8de47d52e32ec8f34a080d08e1e60fd3c9). Tested on my machine. No need to add anything in `.spacemacs`.

I am closing the issue but let me know if you find any issues and I will reopen.
